[IMP] iot: add ip address tips (#18035)
odoo/documentation: content/applications/general/iot/iot_box.rst

#. :doc:`Connect the IoT box to your Odoo database <connect>`.

.. seealso::
   - `Video: How to Set Up and Use the Odoo IoT Box: A Beginner's Guide

>      <https://www.youtube.com/watch?v=w2_Dcm3r_7o&ab_channel=Odoo>`_

Reviewer: ```suggestion
     <https://www.youtube.com/watch?v=w2_Dcm3r_7o>`_
```
Not necessary to use this attribution tracking.

LED status
==========

The IoT box features two LEDs, located to the right of the SD card slot, that indicate its status
and help with troubleshooting when no display is connected. The LEDs can be interpreted as follows:

> - **Red LED lit**: The IoT box is not connected to Internet. Ensure the Ethernet cable is properly

Reviewer: ```suggestion
- **Red LED lit**: The IoT box is not connected to the Internet. Ensure the Ethernet cable is properly
```

IoT box homepage
================

To access the IoT box's homepage on a device, first ensure the device is on the same network as the
IoT box. Then, if using the IoT app, click :guilabel:`Homepage`, or open a browser on the device and
navigate to the IoT box's IP address.

.. image:: iot_box/iot-homepage.png
   :scale: 75%
   :alt: IoT box's homepage

If the IP address is not already known, use any of the following methods to retrieve it:

- While connected to the same network as the IoT box, open a web browser and navigate to
  `http://iotbox.local`. The page automatically redirects to the homepage, which displays the IP
  address.
- Connect a `supported receipt or label printer <https://www.odoo.com/app/iot-hardware>`_ using a
  USB cable, which then automatically prints a ticket with the IP address.
- Connect the IoT box to an :doc:`external monitor <devices/screen>`, which then displays the IP
  address on the screen.

> - On IoT box images `L2026.05.25` and later, the IoT Box IP address is displayed as a discovered

Reviewer: ```suggestion
- On IoT box images `L2026.05.25` and later, the IoT box IP address is displayed as a discovered
```

>   Bluetooth device for 5 minutes after the boot.

Reviewer: ```suggestion
- On IoT box images `L2026.05.25` and later, the IoT Box IP address is displayed as a discovered
  Bluetooth device for 5 minutes after booting.
```

- Access the router's administrator interface, or use third-party software to scan the network.

   .. image:: iot_box/iot-status-display.png
      :scale: 75%
      :alt: Status display with IoT box's IP address

.. tip::
   - To manually restart the IoT box, access the :ref:`homepage <iot/iot-box/homepage>`, then click
     the :icon:`fa-power-off` :guilabel:`(power off)` button in the top-right corner.
   - If the IoT box is connected to an :doc:`external monitor <devices/screen>` or a :doc:`printer
     <devices/printer>`, its homepage can also be accessed by scanning the displayed or printed QR
     code with a device connected to the same network as the IoT box.

>    - To fully reboot the IoT box, unplug and plug it back in.

Reviewer: ```suggestion
   - To fully reboot the IoT box, unplug it and plug it back in.
```
